scripts/bash/run_enceladus.sh

```diff
@@ -231,9 +240,26 @@ if [[ -z "$DRY_RUN" ]]; then
     DATE=`date +%Y_%m_%d-%H_%M_%S`
     NAME=`sed -e 's#.*\.\(\)#\1#' <<< $CLASS`
     TMP_PATH_NAME="$LOG_DIR/enceladus_${NAME}_${DATE}.log"
+
+    # Initializing Kerberos ticket
+    if [[ ! -z "$MENAS_AUTH_KEYTAB" ]]; then
+      # Get principle stored in the keyfile
+      PR=`printf "read_kt $MENAS_AUTH_KEYTAB\nlist" | ktutil | sed -n '5p' | awk '{print $3}' | cut -d '@' -f1`
```

Comment:
Is the `sed -n '5p'` failsafe way to go? on my mac, it seems it is on the 4th line. 
But I am not sure if a failsafe way exists. Could the bellow work on a larger scale? 
```bash
printf "read_kt $MENAS_AUTH_KEYTAB\nlist" | grep -Pio "(?<=\ )[A-Za-z0-9]*?(?=@) | head -1" 
```

Next thing is, do we want to throw an error if PR is empty?